### PR TITLE
Feature/ab#645 integrate new version of keycloak

### PR DESCRIPTION
**Pull request by DerCed**

### CODE REVIEW

**Comment by DerCed** [comment]
Missing: Realm in configmap

**Review by witchpou — CHANGES_REQUESTED** [review]
It needs to be clarify, why readiness, startup and liveness probes are deleted.

File: charts/cityos/Chart.yaml [hunk 1]
```diff
@@ -24,9 +24,10 @@ appVersion: "develop_v2"
 
 dependencies:
   - name: keycloak
-    version: 18.1.1
-    repository: https://codecentric.github.io/helm-charts
```
Comment (witchpou) [inline charts/cityos/Chart.yaml:28]: sehr schön

**Comment by DerCed** [comment]
> It needs to be clarify, why readiness, startup and liveness probes are deleted.

Is enabled by default by the keycloak helm chart:
https://artifacthub.io/packages/helm/bitnami/keycloak

image

**Review by witchpou — APPROVED** [review]

**Pull request merged**